amd/src/app/block/element.js

```diff
@@ -252,9 +252,14 @@ export default class BlockElement {
             checkbox?.classList?.remove('d-none');
         }
 
-        this.#element.querySelector('.no-items')?.remove();
+        this.#element.querySelector('.no-items').classList.add('d-none');
```

Comment:
Since '?' was used before, are we sure it's always found, so we don't get an error in the console?

Reply:
Yup, i made it so it's no longer conditional and instead has a conditional class `d-none`